src/stactools/esa_worldcover/commands.py

```diff
@@ -31,15 +31,23 @@ def esaworldcover():
                   default="esa-worldcover",
                   show_default=True,
                   help="collection id string")
-    def create_collection_command(infile: str, outdir: str, id: str) -> None:
-        """Creates a STAC Collection for Items defined by the hrefs in INFILE"
+    @click.option("-q",
+                  "--quality_assets",
```

Comment:
```suggestion
                  "--include-quality-assets",
```

I think it scans weird without the "include" in the name...feel free to ignore me though if you like how it is :-)

Reply:
Not a bad thought. Using "include" makes it clear. Will update.

Reply:
updated